src/main/java/frc/utils/devicewrappers/SparkMaxDeviceID.java

```diff
@@ -0,0 +1,11 @@
+package frc.utils.devicewrappers;
+
+import com.revrobotics.CANSparkLowLevel;
+
+public record SparkMaxDeviceID(int ID, CANSparkLowLevel.MotorType type) {
+
+	public SparkMaxDeviceID(int ID) {
```

Comment:
how are people supposed to know it is only for brushless

Reply:
@drorta